### DISCUSSION TITLE
Android/IOS

### DISCUSSION THREAD

**Discussion by Cutenekolover** [zippity21/Stardew_Valley_Regression_Mod#60] [General]
I've been scrolling through this trying to find anyway I can get this for Android. Not having any coding experience I can't move it to mobile. Is there any possibility it will be released for Android/IOS?

**Comment by zippity21** [comment]
> I've been scrolling through this trying to find anyway I can get this for Android. Not having any coding experience I can't move it to mobile. Is there any possibility it will be released for Android/IOS?

Unfortunately, this mod is not available on mobile. I haven't looked into what it would take to port it. I have a feeling that it'd have to be almost entirely rewritten, but I definitely could be wrong.

I can try looking into it when I get a chance, or if anyone else wants to attempt it.

--- Edit ---
I did a really quick search and while I still don't know how much effort it would be to port this mod, I found a larger issue: the mobile versions are still on v1.4. This mod is only compatible with 1.5+ (and I have no intention of releasing a version vor any version prior, due to technical issues).

I can definitely look more into it when 1.5 is released on mobile.
